testing/game/Game004 - solve cypher.md

```diff
@@ -4,10 +4,10 @@
 >this testcase verifies that the team can solve the cypher with correct codeword
 
 *Preconditions*
-* There are at least two cyphers created (cypher1 with value = 25 and cypher2 with value = 10)
+* There are at least two cyphers created (cypher1 with value = 10 and cypher2 with value = 10)
```

Comment:
Za mě bych ty body tady asi vynechal. Protože nejde vytvořit šifru s konkrétním bodovým ohodnocením. Ta je fixní.

Reply:
souhlas